composer.json

```diff
@@ -28,18 +28,18 @@
     }
   ],
   "require": {
-    "php": ">=7.0.8",
     "ckeditor/panelbutton": "^4.10",
+    "composer/installers": "^1.2",
```

Comment:
This one is dependency od recommended, also it is under fire in https://www.drupal.org/project/drupal/issues/3134648

Reply:
Removed as it is dependency of another package